### ISSUE TITLE
Error on running the CSV based example

### ISSUE THREAD

**Issue by cgravill** [CSBiology/FSharpGephiStreamer#10]
**Describe the bug**
Error on running the CSV based example

**To Reproduce**

- Follow the install instructions
- Obtain the CSVs from https://github.com/CSBiology/FSharpGephiStreamer/tree/master/docsrc/content/data
- Take the https://csbiology.github.io/FSharpGephiStreamer/exampleAnalysis.html

```fsharp
open System.IO
open System.Text

let readFromFile (file:string) =
        seq {use textReader = new StreamReader(file, Encoding.Default)
             while not textReader.EndOfStream do
                 yield textReader.ReadLine()}

let nodes = 
    readFromFile (__SOURCE_DIRECTORY__ + "/data/goNodeList.csv") 
    |> List.ofSeq
    |> List.map (fun n -> let tmp = n.Split([|','|])
                          createGONode tmp.[0] tmp.[1] tmp.[2] (Colors.Table.StatisticalGraphics24.getRandomColor()))

let edges = 
    readFromFile (__SOURCE_DIRECTORY__ + "/data/goEdgeList.csv") |> List.ofSeq
    |> List.ofSeq
    |> List.map (fun n -> let tmp = n.Split([|','|])
                          createGOEdge (tmp.[0] |> int) tmp.[1] tmp.[2] (nodes |> List.find(fun n -> n.Id = tmp.[2])).Color)
```
Run the code

```
System.FormatException
  HResult=0x80131537
  Message=Input string was not in a correct format.
  Source=System.Private.CoreLib
  StackTrace:
   at System.Number.ThrowOverflowOrFormatException(ParsingStatus status, TypeCode type)
   at System.Number.ParseInt32(ReadOnlySpan`1 value, NumberStyles styles, NumberFormatInfo info)
   at System.Int32.Parse(String s, NumberStyles style, IFormatProvider provider)
   at Microsoft.FSharp.Core.LanguagePrimitives.ParseInt32(String s)
```

Inspecting the files, it looks like you need to skip the first row as you have a header there whereas the parsing code expects just data.

**Expected behavior**
Parse the pre-supplied edges and node

**OS and framework information (please complete the following information):**
 - OS: Windows 10
 - .Net core SDK version: 2.2.203

https://github.com/openjournals/joss-reviews/issues/1445

**Comment by kMutagene** [comment]
Hi @cgravill , 

first of all thank you and @PrashantVaidyanathan for taking the time to review this submission.

You are correct, the first line should be skipped because those are the headers. I was able to reproduce the error and it is located in parsing the edge list. I will fix that asap.

**Comment by kMutagene** [comment]
Should be fixed by #11
